frame/election-provider-support/src/onchain.rs

```diff
@@ -101,6 +101,7 @@ impl<T: Config> ElectionProvider<T::AccountId, T::BlockNumber> for OnChainSequen
 #[cfg(test)]
 mod tests {
 	use super::*;
+	use crate::ConstU32;
```

Comment:
```

Reply:
Removed, and fully qualified the use of `ConstU32`